public/index.html

```diff
@@ -49,4 +49,11 @@
     -->
 </body>
 
+<!-- JavaScript Bundle with Popper -->
```

Comment:
what's this do?

Reply:
Good question. I did not even notice this.

It is some Bootstrap Automatisations. Will check them out and see if I can remove them..